src/main/java/scm/address/logic/commands/ImportCommand.java

```diff
@@ -132,6 +154,67 @@ public List<JsonAdaptedPerson> readPersons(File file)
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Reads the csv data inside {@code file} and returns a List of JsonAdaptedPersons.
+     *
+     * @param file A csv file.
+     * @return A List of JsonAdaptedPerson present inside the {@code file}.
+     * @throws DataLoadingException If the {@code file} is unable to be loaded.
+     */
+    private List<JsonAdaptedPerson> readPersonsFromCsv(File file) throws DataLoadingException {
+        String filePath = file.getPath();
+        List<JsonAdaptedPerson> persons = new ArrayList<>();
+        try {
+            BufferedReader br = new BufferedReader(new FileReader(filePath));
+            String line = "";
+            String splitBy = ",(?=([^\"]*\"[^\"]*\")*[^\"]*$)";
+            String headers = br.readLine();
+            while ((line = br.readLine()) != null) {
+                String[] info = line.split(splitBy);
+                String name = info[0];
+                String phone = info[1];
+                String email = info[2];
+                String address = info[3];
+                address = address.replaceFirst("^\"", "");
+                address = address.replaceFirst("\"$", "");
+
+                // If there are no tags, add the person without tags
+                if (info.length < 5) {
+                    persons.add(new JsonAdaptedPerson(name, phone, email, address, new ArrayList<JsonAdaptedTag>()));
+                    continue;
+                }
+
+                List<JsonAdaptedTag> tags = Arrays.stream(info[4].split(" \\| "))
+                        .map(String::trim)
+                        .map(JsonAdaptedTag::new)
+                        .collect(Collectors.toList());
+
+                JsonAdaptedPerson person = new JsonAdaptedPerson(name, phone, email, address, tags);
+                persons.add(person);
+            }
+        } catch (FileNotFoundException e) {
+            throw new DataLoadingException(new Exception("Cannot load file: " + file.getPath()));
```

Comment:
The message should be a constant, I think?

Reply:
Noted!